### DISCUSSION TITLE
Black screens are showing up with Red LEDs

### DISCUSSION THREAD

**Discussion by 3Huskies** [awawa-dev/HyperHDR#703] [General]
New user here so I am getting familiar with HyperHDR so thank you in advance for any help.

Setup using a Ultra 4K Capture usb 3.0 HD Capture Card.
LED strip: WS2812B
LED Hardware: WS281X
Powered 5v 10A brick 
Raspberry Pi 4 

The colors seem to work fine - however during Plex and when I watch anything from Youtube - and blacks appear, the LEDs will show red. 

I was able to calibrate to 100%. Here is my config. I am still trying to understand it all but I am installing a second instance on my bedroom TV once I get the power brick. 
[HyperHDR-19.0.0.0-Backup-2024-01-02.json](https://github.com/awawa-dev/HyperHDR/files/13812660/HyperHDR-19.0.0.0-Backup-2024-01-02.json)

I have searched the discussion boards and I am seeing that someone has seen green LEDs show when screens are black but I tried making setting the hardware brightness control to -1. 
![shot102532](https://github.com/awawa-dev/HyperHDR/assets/155475185/ef476b9c-dec6-4472-a3de-17d60909f5b9)

What's odd is that the black screen is showing but in the preview it's showing red as seen in the screen shot. However this is only on a YouTube video. In a Prime Video the colors show correct. I am stumped. 
![shot102713](https://github.com/awawa-dev/HyperHDR/assets/155475185/dd49e16f-f176-4182-b70a-15751d69068d)

**Comment by awawa-dev** [comment]
Hi
First please read the FAQ in the Wiki about your LED driver.

Is this what shows up in you HyperHDR live preview windows?
<img src="https://private-user-images.githubusercontent.com/155475185/293758821-ef476b9c-dec6-4472-a3de-17d60909f5b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyMjEyMDksIm5iZiI6MTcwNDIyMDkwOSwicGF0aCI6Ii8xNTU0NzUxODUvMjkzNzU4ODIxLWVmNDc2YjljLWRlYzYtNDQ3Mi1hM2RlLTE3ZDYwOTA5ZjViOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwMlQxODQxNDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZjQyNzk5YmU0ZDdjNWZkM2NjODAwMTZjNDRhYjJkZDQxMDE5MTRjMDQ2YmQyMGE5MDg1N2Y5ZjE0YjFlYTdkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.w_7ukRKuK7Hx7eZvPS6kUcoA4cfv1LTMImdYKWS7eCc"/>
or this: ``` however during Plex and when I watch anything from Youtube - and blacks appear``` I suspect HDCP but then it's your fault that you agreed to use apps that force you to accept HDCP ;) And HDCP (DRM) has serious consequences that now make your live harder.

But to summarize: is HyperHDR live video streaming black or red? Are the LEDs bright red or just dim?

**Comment by 3Huskies** [comment]
It’s mainly when the black letter boxes/screen is black and they are bright red. It’s always fun to have a challenge it’s better understanding it and knowing what’s going on then just plugging it in and working.

**Comment by awawa-dev** [comment]
If you have all red screen in the live preview but only from time to time (eg. in specific conditions like you described) for the same video source/movie then probably something is wrong with the USB capture card, its power supply or HDMI cables: it's hard to guess. There is no error in the HyperHDR logs?

**Comment by 3Huskies** [comment]
Figured it out. After playing with every cable and alternating out resources here was my solution and thank you for helping me isolate the problem:

1) remove DVR from equation
2) plug in firestick directly to tv to isolate cable issues if any
3) isolated that the dvr was causing the issues because swapping out parts nothing worked.
4) bought an hdcp 2.3 splitter and split the signal from the firestick before it hits the dvr going directly to the capture card. 
5) take the second split cable and run the normal way from firestick to dvr to tv for sound. The signal came out normal after doing calibration and letterbox is working successfully as well.


I did do my second setup immediately and it worked right out the box with the exception of a low voltage issue that was due to the power block. 

Thanks again for helping me isolate the issue. I love this setup!!! Many thanks for your development on such an awesome product and being able to at least offer help!
